src/ziggurat/java/utils/ziggurat_util.clj

```diff
@@ -0,0 +1,62 @@
+(ns ziggurat.java.utils.ziggurat-util
```

Comment:
Can we have a different name for this ns?

Reply:
changed